docs/contributing/how-to-open-a-pull-request.md

```diff
@@ -1,135 +1,135 @@
 ---
-title: How to Open a Pull Request
+title: Como Abrir uma Pull Request
```

Comment:
Acho que é mais comum o artigo "um Pull Request" ao invés "uma Pull Request". Eu pelo menos conversando em conversas de IT sempre falei "um".

Reply:
Hey @Dennissiq, eu concordo com você, também utilizo mais "um Pull Request" ao invés "uma Pull Request". Porém eu tomei essa decisão pois encontrei escrito dessa maneira em outros lugares aqui na doc: 

> abri uma Pull Request 
> revise a PR
> como abrir uma PR
https://github.com/gatsbyjs/gatsby-pt-BR/blob/master/CONTRIBUTING.md#finalizei-a-tradu%C3%A7%C3%A3o

Além de encontrar da mesma maneira nas docs do GitHub:

> As pull requests
https://help.github.com/pt/github/collaborating-with-issues-and-pull-requests/about-pull-requests

Perante isso, devo realmente alterar de "a/uma PR" para "o/um PR"?

Reply:
Done: https://github.com/gatsbyjs/gatsby-pt-BR/pull/29/commits/3b3c21095fb45f6be256aa235cc1b8677bc185fa

Reply:
ah ok, pode deixar assim então ;) 
Acho que os dois podem ser válidos, como se trata de uma palavra em inglês acho que blz